scripts/live_counts.py

```diff
@@ -0,0 +1,127 @@
+
+* Maintainers
+* Number of people in core (active, emeritus)
+* Number of staged-recipes reviewers
+* Number of repositories (and feedstocks)
+* Number of output packages (a feedstock can produce more than one package name)
+* Number of artifacts (a package name will have several builds)
+* Number of supported platforms
+* Number of total commits in the organization (number of commits made by bots)
+* Number of issues and PRs
+* Total storage used at Anaconda.org
+* Number of downloads and bandwidth used
+* Number of CI runs
+"""
+
+import os
+import json
+
+from github import Github
+import requests
+
+GH_TOKEN = "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" # os.environ["GITHUB_TOKEN"])
+
+
+def _gh_api_query_total_count(query, **kwargs):
+    r = requests.get(f"https://api.github.com/{query}",
+        headers={"Authorization": f"token {GH_TOKEN}"},
+        **kwargs
+    )
+    r.raise_for_status()
+    data = r.json()
+    try:
+        return data["total_count"]
+    except KeyError:
+        print("ERROR!\n", data)
+        raise
+
+
+def github_data():
+    print("Initializing for Github...")
+
+    gh = Github(GH_TOKEN)
+    org = gh.get_organization("conda-forge")
+    # get creation date
+    print("Getting creation date...")
+    created = str(org.created_at)
+
+    # get number of members
+    print("Getting members...")
+    n_members = org.get_members().totalCount
+    members_core = set(m.login for m in org.get_team_by_slug("Core").get_members())
+    n_members_core = len(members_core)
+    members_staged_recipes = set(m.login for m in org.get_team_by_slug("staged-recipes").get_members())
+    n_members_staged_recipes = sum(1 for login in members_staged_recipes if login not in members_core)
+
+    # get number of issues and PRs - we need to use requests directly here, pygithub says totalCount is 1000 ??
+    print("Getting issues / PRs...")
+    n_open_issues = _gh_api_query_total_count("search/issues", params={"q": "org:conda-forge is:issue is:open"})
+    n_closed_issues = _gh_api_query_total_count("search/issues", params={"q": "org:conda-forge is:issue is:closed"})
+    n_open_prs = _gh_api_query_total_count("search/issues", params={"q": "org:conda-forge is:pr is:open"})
+    n_closed_prs = _gh_api_query_total_count("search/issues", params={"q": "org:conda-forge is:pr is:closed"})
+
+    return {
+        "created_at": created,
+        "n_members": n_members,
+        "n_members_core": n_members_core,
+        "n_members_staged_recipes": n_members_staged_recipes,
+        "n_repos": org.public_repos,
+        "n_issues": n_open_issues + n_closed_issues,
+        "n_open_issues": n_open_issues,
+        "n_closed_issues": n_closed_issues,
+        "n_prs": n_open_prs + n_closed_prs,
+        "n_open_prs": n_open_prs,
+        "n_closed_prs": n_closed_prs,
+    }
+
+
+def libcgfraph_data():
+    print("Getting artifacts...")
+    url = "https://github.com/regro/libcfgraph/raw/master/.file_listing.json"
```

Comment:
We may want to use the anaconda API since libcfgraph can be behind here.

Reply:
```
curl -X 'GET' \
  'https://api.anaconda.org/packages/conda-forge' \
  -H 'accept: application/json'
```

results in 504 😬 

I'll see if there's a leaner way to query the API. Otherwise, scraping this number here might be enough:

<img width="450" alt="image" src="https://user-images.githubusercontent.com/2559438/162492810-d1b2ddd5-f09f-4541-9aef-5a94d2655aa2.png">

Alternatively, collect ALL repodatas (or maybe channeldata?) and count?

Reply:
All of the repodata is a great idea! That should be fast enough and easy to code up!

Reply:
I think this will work

```
print(len(requests.get("https://conda.anaconda.org/conda-forge/channeldata.json").json()["packages"]))
```

Reply:
This will get you the number of platforms too.

Reply:
passing on this since it includes defunct platforms and packages.